### PR TITLE
Fix broken link in docker readme

### LINKED ISSUE #14221: Bug Report: Broken link in docker readme
### Overview of the Issue

The third bullet point under the "Principles" heading links to [lite/build.sh](https://github.com/vitessio/vitess/blob/main/docker/lite/build.sh), which does not exist. 

### Reproduction Steps

N/A

### Binary Version

```sh
main branch (commit e5eea9c)
```

### Operating System and Environment details

```sh
N/A
```

### Log Fragments

_No response_

### PR DESCRIPTION
**Pull request by ravicodelabs**
## Description

This is a fix of a broken link in the [docker readme](https://github.com/vitessio/vitess/blob/main/docker/README.md). In particular, the third bullet under the "Principles" heading links to [lite/build.sh](https://github.com/vitessio/vitess/blob/main/docker/lite/build.sh, which does not exist, and is therefore substituted with another `build.sh` file link that works. 

## Related Issue(s)

Fixes #14221 

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required

### CODE REVIEW

**Review by frouioui — APPROVED** [review]
This looks good to me, thank you! However, the DCO check is failing, can you please amend your commit and sign it.

**Comment by ravicodelabs** [comment]
> This looks good to me, thank you! However, the DCO check is failing, can you please amend your commit and sign it.

Done!

**Comment by frouioui** [comment]
Hey @ravicodelabs it seems like the DCO check is still failing. Please follow the instructions on https://github.com/vitessio/vitess/pull/14222/checks?check_run_id=17579049266.

**Comment by ravicodelabs** [comment]
Hi @frouioui, sorry for the back and forth! I've added the Signed-off-by line as well, so hopefully it should go through this time.

**Comment by frouioui** [comment]
> Hi @frouioui, sorry for the back and forth! I've added the Signed-off-by line as well, so hopefully it should go through this time.

It went through! Thank you 🙏🏻 We will get you a second review, and then we can merge this.

**Comment by frouioui** [comment]
I force pushed to your branch @ravicodelabs to re-trigger CI as it was stuck for some reason.

**Review by deepthi — APPROVED** [review]

**Comment by frouioui** [comment]
Done @ravicodelabs, thank you for your contribution! 🚀

**Pull request merged**
